packages/icons/icons.json

```
@@ -4198,5 +4198,12 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Clipboard file",
```

I think variations of an icon usually use a colon for the display name (e.g. `Cloud: upload`)
```suggestion
        "displayName": "Clipboard: file",
```